From 9d857d1871b1c9e3b988412d691e9dfba4a14f6d Mon Sep 17 00:00:00 2001
From: Alf Gaida <agaida@siduction.org>
Date: Wed, 29 Nov 2017 01:00:07 +0100
Subject: [PATCH] move panel config to /usr/share/lxqt

---
 panel/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panel/CMakeLists.txt b/panel/CMakeLists.txt
index 5a322a3e..522c1ffc 100644
--- a/panel/CMakeLists.txt
+++ b/panel/CMakeLists.txt
@@ -114,7 +114,7 @@ target_link_libraries(${PROJECT}
 )
 
 install(TARGETS ${PROJECT} RUNTIME DESTINATION bin)
-install(FILES ${CONFIG_FILES} DESTINATION ${LXQT_ETC_XDG_DIR}/lxqt)
+install(FILES ${CONFIG_FILES} DESTINATION ${CMAKE_INSTALL_DATADIR}/lxqt)
 install(FILES ${PUB_HEADERS} DESTINATION include/lxqt)
 install(FILES
     man/lxqt-panel.1
